### PR TITLE
Initialisation du projet & Creation du design du Navbar

### PR DESCRIPTION
**Pull request by jakemelvin**
J'ai conçu le projet en Next et j'ai créé le component pour le navbar en gérant également le responsive de celui-ci

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
```
Comment (libock-serge) [inline tsconfig.json]: Hello, please set the  "allowJs" attribute here to false

Comment (jakemelvin) [reply tsconfig.json]: Okay understood

File: src/styles/navbar.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+.app__navbar-mode {
+  display: flex;
+  gap: 0.5rem;
+  align-items: center;
+  cursor: pointer;
+}
+/*Gestion du responsive*/
+@media screen and (max-width: 450px) {
+  .app__navbar {
+    padding: 1rem 1rem;
```
Comment (yannmarc) [inline src/styles/navbar.css:24]: Salut @jakemelvin consistency in your css units, I see you use rem's and some other places px. A good ideas will be to define a mixin that converts your px's to rem's

```
@function rem($pixels, $base-font-size: 16px) {
  @return ($pixels / $base-font-size) * 1rem;
}
```

Comment (jakemelvin) [reply src/styles/navbar.css:24]: So I either have to use pixel or rem is that it? Even in the max-width of my media query?

Comment (jakemelvin) [reply src/styles/navbar.css:24]: I made the differents changes to the unit used, to have the same units everywhere I also change the units in the different font sizes to use em

**Review by libock-serge — APPROVED** [review]

**Pull request merged**